src/actions/articles/articles.js

```diff
@@ -0,0 +1,73 @@
+import { articlesType } from '../../actions-types';
```

Comment:
@djallas, it would be better if you put one action in its own respective file then export the file in `index.js`, that would a good practice of modularity